GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -325,7 +362,34 @@ public void setRenderingOrder(int renderingOrder) {
      *         not.
```

Comment:
make sure to mark the methods as deprecated with the @deprecated tag.  See setKeepWrapper() in GVRHybridObject.java as an example.